client/src/app/tabs/bpmn/BpmnEditor.js

```diff
@@ -301,20 +303,31 @@ export class BpmnEditor extends CachedComponent {
 
   handleImport = (error, warnings) => {
     const {
+      isNew,
       onImport,
       xml
     } = this.props;
 
+    let {
```

Comment:
```suggestion
    const {
```
It's not modified anywhere.